tests/vmi_lifecycle_test.go

```diff
@@ -1157,11 +1159,13 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 
 			})
 
-			It("the vmi with EVMCS HyperV feature should have correct hyperv and cpu features auto filled", func() {
+			DescribeTable("the vmi with EVMCS HyperV feature should have correct hyperv and cpu features auto filled", func(enabled bool) {
```

Comment:
Can you please change the input argument to `FeatureState` and add a case with an empty state? This would help us keep compatibility.

Reply:
Can you please elaborate what do you mean by empty state?
I don't see how FeatureState is better than enabled..

Reply:
Sure, I would like to see 3 entries as:
`FeatureState{}`
`FeatureState{Enabled: true}`
`FeatureState{Enabled: false}`

Reply:
ok Done